internal/provider/resource_settings_risk_engine.go

```
@@ -0,0 +1,742 @@
+		"enabled":     types.BoolType,
+		"rules":       types.ListType{ElemType: riskBlockableRuleObjectType},
+		"filter_expr": types.StringType,
+	}
+	riskBlockableRulesObjectType = types.ObjectType{AttrTypes: riskBlockableRulesAttrTypes}
+
+	riskLocalPluginsAttrTypes = map[string]attr.Type{
+		"virus_total":     riskVirusTotalObjectType,
+		"reversing_labs":  riskReversingLabsObjectType,
+		"blockable_rules": riskBlockableRulesObjectType,
+	}
+	riskLocalPluginsObjectType = types.ObjectType{AttrTypes: riskLocalPluginsAttrTypes}
```

golangci-lint (2.12.2)

[error] var riskLocalPluginsObjectType is unused

(unused)